### DISCUSSION TITLE
Can two nodes with two drives per node form a distributed cluster?

### DISCUSSION THREAD

**Discussion by Hello-Linux** [minio/minio#19430] [Q&A] [answered]
The first question ：We have two local node servers with only two disks on each server. We would like to know whether we can form a distributed cluster with erasure codes?

And the second question： In minio doc I see the [config](https://min.io/docs/minio/linux/reference/minio-server/settings/storage-class.html#environment-variables)
<img width="659" alt="000" src="https://github.com/minio/minio/assets/17851645/b7ff999d-6cb7-4c0a-890f-89e876fe466d">

Now I have two nodes，every node have one disk drive，and I set Erasure Set Size=2，and EC:1，Is the total result a data block on one disk and a check block on another disk?If so, then one of my two nodes is down, can the whole cluster still be used?

**Comment by harshavardhana** [comment, marked as answer]
You can but not recommended for any serious workload.

**Reply by Hello-Linux** [reply]
Hello, thank you for your answer. For question 1, two nodes, two disks per node, what is the risk？

**Reply by Hello-Linux** [reply]
For my second question, am I expressing it correctly?

**Reply by harshavardhana** [reply]
Risk is no HA

**Reply by Hello-Linux** [reply]
for my first question，I used nginx to proxy these two minio nodes, I shut down one of the nodes, my cluster can still work normally, why there is no risk of ha?

**Reply by Hello-Linux** [reply]
And the second question： In minio doc I see the [config](https://min.io/docs/minio/linux/reference/minio-server/settings/storage-class.html#environment-variables)

Now I have two nodes，every node have one disk drive，and I run mc admin info alias command  ,I see :
```
●  minio-1.testcom:9000
   Uptime: 24 seconds 
   Version: 2024-03-15T01:07:19Z
   Network: 2/2 OK 
   Drives: 1/1 OK 
   Pool: 1

●  minio-2.test.com:9000
   Uptime: 1 hour 
   Version: 2024-03-15T01:07:19Z
   Network: 2/2 OK 
   Drives: 1/1 OK 
   Pool: 1

Pools:
   1st, Erasure sets: 1, Drives per erasure set: 2
```
I have two disk，the Erasure sets should eqal 2，why 1？and I wonder how the data blocks and parity blocks are distributed on these two tapes in this case, one disk storing the data block and one disk storing the parity block? Or is it both sides?
